src/i18n/locale/en.json

```diff
@@ -27,6 +27,12 @@
       "set_categories": "Set categories"
     }
   },
+  "categories": {
```

Comment:
I would add this under `"category"` as `"settings"`

```
"category": {
  ...
  "settings": {
    "inclusion": {
      "label": {
        "exclude": "Exclude: {{excludedCategoriesText}}",
        "include": "Include: {{includedCategoriesText}}"
      }
    }
}
```

Reply:
Done